sdk/provisioning/Generator/src/Model/SimpleModel.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Azure.Provisioning.Generator.Model;
+
+public class SimpleModel(Specification spec, Type armType, string name, string? ns = default, string? description = default)
+    : TypeModel(spec, armType, name, ns, description)
```

Comment:
You can skim through this class if you want to get an idea of how easy it is to add new generated code.  This is dumping out our simple `ProvisioningConstruct` models like `ManagedServiceIdentity` we saw earlier.